File: add/codesnippet/CSharp/m-system.net.networkinfo_9_1.cs
         public static void ShowActiveTcpConnections()
         {
                    Console.WriteLine("Active TCP Connections");
                    IPGlobalProperties properties = IPGlobalProperties.GetIPGlobalProperties();
                    TcpConnectionInformation[] connections = properties.GetActiveTcpConnections(); 
                    foreach (TcpConnectionInformation c in connections)
                    {
                        Console.WriteLine("{0} <==> {1}", 
                            c.LocalEndPoint.ToString(), 
                            c.RemoteEndPoint.ToString());
                    }
         }

File: add/codesnippet/CPP/m-system.net.networkinfo_9_1.cpp
void ShowActiveTcpConnections()
{
   Console::WriteLine( "Active TCP Connections" );
   IPGlobalProperties ^ properties = IPGlobalProperties::GetIPGlobalProperties();
   array<TcpConnectionInformation^>^connections = properties->GetActiveTcpConnections();
   System::Collections::IEnumerator^ myEnum6 = connections->GetEnumerator();
   while ( myEnum6->MoveNext() )
   {
      TcpConnectionInformation ^ c = safe_cast<TcpConnectionInformation ^>(myEnum6->Current);
      Console::WriteLine( "{0} <==> {1}", c->LocalEndPoint, c->RemoteEndPoint );
   }
}